### PR TITLE
CS fixes for Block folder

### PR DESCRIPTION
**Pull request by simon-letch**

### CODE REVIEW

File: Block/Wishlist.php [hunk 1]
```diff
@@ -107,7 +111,7 @@ public function getMode()
     /**
      * Product url.
      *
-     * @param $store
+     * @param string $store
```
Comment (cdiacon) [inline Block/Wishlist.php]: $store should be int

File: Block/Adminhtml/Config/Customdatafields.php [hunk 2]
```diff
@@ -150,7 +155,7 @@ public function _prepareArrayRow(\Magento\Framework\DataObject $row)
     public function _getAttributeRenderer()
     {
         $this->attributeRenderer = $this->getLayout()->createBlock(
-            'Dotdigitalgroup\Email\Block\Adminhtml\Config\Select',
+            Dotdigitalgroup\Email\Block\Adminhtml\Config\Select::class,
```
Comment (adeelq) [inline Block/Adminhtml/Config/Customdatafields.php]: need to be \Dotdigitalgroup\Email\Block\Adminhtml\Config\Select::class

File: Block/Adminhtml/Config/Customdatafields.php [hunk 3]
```diff
@@ -166,7 +171,7 @@ public function _getAttributeRenderer()
     public function _getDatafieldRenderer()
     {
         $this->datafieldRenderer = $this->getLayout()->createBlock(
-            'Dotdigitalgroup\Email\Block\Adminhtml\Config\Select',
+            Dotdigitalgroup\Email\Block\Adminhtml\Config\Select::class,
```
Comment (adeelq) [inline Block/Adminhtml/Config/Customdatafields.php]: need to be \Dotdigitalgroup\Email\Block\Adminhtml\Config\Select::class

File: Block/Adminhtml/Config/Developer/Automapdatafields.php [hunk 4]
```diff
@@ -37,7 +37,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
         $url = $this->_urlBuilder->getUrl('dotdigitalgroup_email/run/automapdatafields', $params);
 
         return $this->getLayout()
-            ->createBlock('Magento\Backend\Block\Widget\Button')
+            ->createBlock(Magento\Backend\Block\Widget\Button::class)
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Automapdatafields.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Catalogsync.php [hunk 5]
```diff
@@ -35,7 +35,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
         $url = $this->escapeUrl($this->_urlBuilder->getUrl('dotdigitalgroup_email/run/catalogsync'));
 
         return $this->getLayout()
-            ->createBlock('Magento\Backend\Block\Widget\Button')
+            ->createBlock(Magento\Backend\Block\Widget\Button::class)
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Catalogsync.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Connect.php [hunk 6]
```diff
@@ -92,7 +92,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
         )) : $url;
 
         return $this->getLayout()->createBlock(
-            'Magento\Backend\Block\Widget\Button'
+            Magento\Backend\Block\Widget\Button::class
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Connect.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Contactsync.php [hunk 7]
```diff
@@ -35,7 +35,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
         $url = $this->escapeUrl($this->_urlBuilder->getUrl('dotdigitalgroup_email/run/contactsync'));
 
         return $this->getLayout()
-            ->createBlock('Magento\Backend\Block\Widget\Button')
+            ->createBlock(Magento\Backend\Block\Widget\Button::class)
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Contactsync.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Deletecontactids.php [hunk 8]
```diff
@@ -36,7 +36,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
             'dotdigitalgroup_email/run/deletecontactids'
         );
 
-        return $this->getLayout()->createBlock('Magento\Backend\Block\Widget\Button')
+        return $this->getLayout()->createBlock(Magento\Backend\Block\Widget\Button::class)
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Deletecontactids.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Importersync.php [hunk 9]
```diff
@@ -35,7 +35,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
         $url = $this->_urlBuilder->getUrl('dotdigitalgroup_email/run/importersync');
 
         return $this->getLayout()
-            ->createBlock('Magento\Backend\Block\Widget\Button')
+            ->createBlock(Magento\Backend\Block\Widget\Button::class)
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Importersync.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Ordersync.php [hunk 10]
```diff
@@ -35,7 +35,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
         $url = $this->_urlBuilder->getUrl('dotdigitalgroup_email/run/ordersync');
 
         return $this->getLayout()
-            ->createBlock('Magento\Backend\Block\Widget\Button')
+            ->createBlock(Magento\Backend\Block\Widget\Button::class)
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Ordersync.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Resetcatalog.php [hunk 11]
```diff
@@ -41,7 +41,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
         $url = $this->_urlBuilder->getUrl('dotdigitalgroup_email/run/catalogreset', $query);
 
         return $this->getLayout()
-            ->createBlock('Magento\Backend\Block\Widget\Button')
+            ->createBlock(Magento\Backend\Block\Widget\Button::class)
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Resetcatalog.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Resetcustomers.php [hunk 12]
```diff
@@ -35,7 +35,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
         $url = $this->_urlBuilder->getUrl('dotdigitalgroup_email/run/customersreset');
 
         return $this->getLayout()
-            ->createBlock('Magento\Backend\Block\Widget\Button')
+            ->createBlock(Magento\Backend\Block\Widget\Button::class)
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Resetcustomers.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Resetorders.php [hunk 13]
```diff
@@ -41,7 +41,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
         $url = $this->_urlBuilder->getUrl('dotdigitalgroup_email/run/ordersreset', $query);
 
         return $this->getLayout()
-            ->createBlock('Magento\Backend\Block\Widget\Button')
+            ->createBlock(Magento\Backend\Block\Widget\Button::class)
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Resetorders.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Resetreviews.php [hunk 14]
```diff
@@ -41,7 +41,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
         $url = $this->_urlBuilder->getUrl('dotdigitalgroup_email/run/reviewsreset', $query);
 
         return $this->getLayout()->createBlock(
-            'Magento\Backend\Block\Widget\Button'
+            Magento\Backend\Block\Widget\Button::class
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Resetreviews.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Resetsubscribers.php [hunk 15]
```diff
@@ -32,10 +32,10 @@ public function setButtonLabel($buttonLabel)
      */
     public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractElement $element)
     {
-        $url = $this->_urlBuilder->getUrl('dotdigitalgroup_email/run/subscribersreset');
+        $url = $this->_urlBuilder->getUrl(dotdigitalgroup_email/run/subscribersreset::class);
 
         return $this->getLayout()
-            ->createBlock('Magento\Backend\Block\Widget\Button')
+            ->createBlock(Magento\Backend\Block\Widget\Button::class)
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Resetsubscribers.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Resetsubscribers.php [hunk 16]
```diff
@@ -32,10 +32,10 @@ public function setButtonLabel($buttonLabel)
      */
     public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractElement $element)
     {
-        $url = $this->_urlBuilder->getUrl('dotdigitalgroup_email/run/subscribersreset');
+        $url = $this->_urlBuilder->getUrl(dotdigitalgroup_email/run/subscribersreset::class);
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Resetsubscribers.php]: path here in getUrl is not a class name. please revert it to what it was.

File: Block/Adminhtml/Config/Developer/Resetwishlists.php [hunk 17]
```diff
@@ -41,7 +41,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
         $url = $this->_urlBuilder->getUrl('dotdigitalgroup_email/run/wishlistsreset', $query);
 
         return $this->getLayout()->createBlock(
-            'Magento\Backend\Block\Widget\Button'
+            Magento\Backend\Block\Widget\Button::class
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Resetwishlists.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Reviewsync.php [hunk 18]
```diff
@@ -35,7 +35,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
         $url = $this->_urlBuilder->getUrl('dotdigitalgroup_email/run/reviewsync');
 
         return $this->getLayout()
-            ->createBlock('Magento\Backend\Block\Widget\Button')
+            ->createBlock(Magento\Backend\Block\Widget\Button::class)
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Reviewsync.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Subscribersync.php [hunk 19]
```diff
@@ -36,7 +36,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
             = $this->_urlBuilder->getUrl('dotdigitalgroup_email/run/subscribersync');
 
         return $this->getLayout()
-            ->createBlock('Magento\Backend\Block\Widget\Button')
+            ->createBlock(Magento\Backend\Block\Widget\Button::class)
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Subscribersync.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Developer/Wishlistsync.php [hunk 20]
```diff
@@ -35,7 +35,7 @@ public function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractEle
         $url = $this->_urlBuilder->getUrl('dotdigitalgroup_email/run/wishlistsync');
 
         return $this->getLayout()
-            ->createBlock('Magento\Backend\Block\Widget\Button')
+            ->createBlock(Magento\Backend\Block\Widget\Button::class)
```
Comment (adeelq) [inline Block/Adminhtml/Config/Developer/Wishlistsync.php]: need to be \Magento\Backend\Block\Widget\Button::class

File: Block/Adminhtml/Config/Fieldset.php [hunk 21]
```diff
@@ -12,7 +12,7 @@ class Fieldset extends \Magento\Config\Block\System\Config\Form\Fieldset
     public function render(\Magento\Framework\Data\Form\Element\AbstractElement $element)
     {
         $block = $this->getLayout()->createBlock(
-            'Dotdigitalgroup\Email\Block\Adminhtml\Config\Configuration\Preview',
+            Dotdigitalgroup\Email\Block\Adminhtml\Config\Configuration\Preview::class,
```
Comment (adeelq) [inline Block/Adminhtml/Config/Fieldset.php]: need to be \Dotdigitalgroup\Email\Block\Adminhtml\Config\Configuration\Preview::class

File: Block/Adminhtml/Config/Report/Wishlist.php [hunk 22]
```diff
@@ -11,7 +11,7 @@ class Wishlist extends \Magento\Config\Block\System\Config\Form\Field
     public $buttonLabel = 'Contact Report';
 
     /**
-     * @param $buttonLabel
+     * @param strin $buttonLabel
```
Comment (adeelq) [inline Block/Adminhtml/Config/Report/Wishlist.php]: typo should be string

File: Block/Adminhtml/Config/Rules/Customdatafields.php [hunk 23]
```diff
@@ -160,7 +165,7 @@ private function _getAttributeRenderer()
         if (!$this->getAttributeRenderer) {
             $this->getAttributeRenderer = $this->getLayout()
                 ->createBlock(
-                    'Dotdigitalgroup\Email\Block\Adminhtml\Config\Select',
+                    Dotdigitalgroup\Email\Block\Adminhtml\Config\Select::class,
```
Comment (adeelq) [inline Block/Adminhtml/Config/Rules/Customdatafields.php]: need to be \Dotdigitalgroup\Email\Block\Adminhtml\Config\Select::class

File: Block/Adminhtml/Config/Rules/Customdatafields.php [hunk 24]
```diff
@@ -179,7 +184,7 @@ private function _getConditionsRenderer()
         if (!$this->getConditionsRenderer) {
             $this->getConditionsRenderer = $this->getLayout()
                 ->createBlock(
-                    'Dotdigitalgroup\Email\Block\Adminhtml\Config\Select',
+                    Dotdigitalgroup\Email\Block\Adminhtml\Config\Select::class,
```
Comment (adeelq) [inline Block/Adminhtml/Config/Rules/Customdatafields.php]: need to be  \Dotdigitalgroup\Email\Block\Adminhtml\Config\Select::class

File: Block/Adminhtml/Config/Rules/Customdatafields.php [hunk 25]
```diff
@@ -198,7 +203,7 @@ private function _getValueRenderer()
         if (!$this->getValueRenderer) {
             $this->getValueRenderer = $this->getLayout()
                 ->createBlock(
-                    'Dotdigitalgroup\Email\Block\Adminhtml\Config\Select',
+                    Dotdigitalgroup\Email\Block\Adminhtml\Config\Select::class,
```
Comment (adeelq) [inline Block/Adminhtml/Config/Rules/Customdatafields.php]: need to be  \Dotdigitalgroup\Email\Block\Adminhtml\Config\Select::class

File: Block/Adminhtml/Dashboard.php [hunk 26]
```diff
@@ -13,6 +13,11 @@ class Dashboard extends \Magento\Backend\Block\Widget\Container
      */
     public $_template = 'dashboard/main.phtml';
 
+    /**
+     * Undocumented function
```
Comment (adeelq) [inline Block/Adminhtml/Dashboard.php]: Undocumented function? Auto generated? Should be more descriptive

File: Block/Adminhtml/Rules/Edit/Tab/Conditions.php [hunk 27]
```diff
@@ -108,7 +108,7 @@ public function _prepareForm()
             'options' => $this->options->toOptionArray(),
         ]);
         $renderer = $this->getLayout()
-            ->createBlock('Dotdigitalgroup\Email\Block\Adminhtml\Config\Rules\Customdatafields');
+            ->createBlock(Dotdigitalgroup\Email\Block\Adminhtml\Config\Rules\Customdatafields::class);
```
Comment (adeelq) [inline Block/Adminhtml/Rules/Edit/Tab/Conditions.php]: need to be \Dotdigitalgroup\Email\Block\Adminhtml\Config\Rules\Customdatafields::class

File: Block/Adminhtml/Rules/Edit/Tab/Main.php [hunk 28]
```diff
@@ -146,7 +146,7 @@ public function _prepareForm()
                 ]
             );
             $renderer = $this->getLayout()->createBlock(
-                'Magento\Backend\Block\Store\Switcher\Form\Renderer\Fieldset\Element'
+                Magento\Backend\Block\Store\Switcher\Form\Renderer\Fieldset\Element::class
```
Comment (adeelq) [inline Block/Adminhtml/Rules/Edit/Tab/Main.php]: need to be \Magento\Backend\Block\Store\Switcher\Form\Renderer\Fieldset\Element::class

File: Block/Customer/Account/Books.php [hunk 29]
```diff
@@ -66,8 +68,8 @@ public function getSaveUrl()
     /**
      * Get config values.
      *
-     * @param $path
-     * @param $website
+     * @param string $path
+     * @param mixed $website
```
Comment (adeelq) [inline Block/Customer/Account/Books.php]: $website is int

File: Block/Edc.php [hunk 30]
```diff
@@ -6,7 +6,7 @@ class Edc extends \Magento\Framework\View\Element\Template
 {
 
     /**
-     * @param $store
+     * @param string $store
```
Comment (adeelq) [inline Block/Edc.php]: $store can be mixed

File: Block/Feefo.php [hunk 31]
```diff
@@ -181,7 +184,8 @@ public function getProductsReview($check = true)
     }
 
     /**
-     * @param $template
+     * @param mixed $template
```
Comment (adeelq) [inline Block/Feefo.php]: type is string.

File: Block/Recommended/Product.php [hunk 32]
```diff
@@ -129,11 +135,11 @@ public function getLoadedProductCollection()
     }
 
     /**
-     * @param $orderItems
-     * @param $mode
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
+     * @param mixed $orderItems
```
Comment (adeelq) [inline Block/Recommended/Product.php]: array

File: Block/Recommended/Product.php [hunk 33]
```diff
@@ -129,11 +135,11 @@ public function getLoadedProductCollection()
...
+     * @param mixed $mode
```
Comment (adeelq) [inline Block/Recommended/Product.php]: string

File: Block/Recommended/Product.php [hunk 34]
```diff
@@ -129,11 +135,11 @@ public function getLoadedProductCollection()
...
+     * @param mixed $productsToDisplayCounter
```
Comment (adeelq) [inline Block/Recommended/Product.php]: int

File: Block/Recommended/Product.php [hunk 35]
```diff
@@ -129,11 +135,11 @@ public function getLoadedProductCollection()
...
+     * @param mixed $limit
```
Comment (adeelq) [inline Block/Recommended/Product.php]: int

File: Block/Recommended/Product.php [hunk 36]
```diff
@@ -129,11 +135,11 @@ public function getLoadedProductCollection()
...
+     * @param mixed $maxPerChild
```
Comment (adeelq) [inline Block/Recommended/Product.php]: int

File: Block/Recommended/Product.php [hunk 37]
```diff
@@ -168,11 +174,13 @@ private function getProductsToDisplay($orderItems, $mode, &$productsToDisplayCou
...
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
-     * @param $recommendedProducts
-     * @param $productsToDisplay
+     * @param mixed $productsToDisplayCounter
+     * @param mixed $limit
+     * @param mixed $maxPerChild
+     * @param mixed $recommendedProducts
+     * @param mixed $productsToDisplay
```
Comment (adeelq) [inline Block/Recommended/Product.php]: array

File: Block/Recommended/Product.php [hunk 38]
```diff
@@ -168,11 +174,13 @@ private function getProductsToDisplay($orderItems, $mode, &$productsToDisplayCou
...
     /**
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
-     * @param $recommendedProducts
-     * @param $productsToDisplay
+     * @param mixed $productsToDisplayCounter
+     * @param mixed $limit
+     * @param mixed $maxPerChild
+     * @param mixed $recommendedProducts
```
Comment (adeelq) [inline Block/Recommended/Product.php]: array

File: Block/Recommended/Product.php [hunk 39]
```diff
@@ -168,11 +174,13 @@ private function getProductsToDisplay($orderItems, $mode, &$productsToDisplayCou
...
 
     /**
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
-     * @param $recommendedProducts
-     * @param $productsToDisplay
+     * @param mixed $productsToDisplayCounter
+     * @param mixed $limit
+     * @param mixed $maxPerChild
```
Comment (adeelq) [inline Block/Recommended/Product.php]: int

File: Block/Recommended/Product.php [hunk 40]
```diff
@@ -168,11 +174,13 @@ private function getProductsToDisplay($orderItems, $mode, &$productsToDisplayCou
     }
 
     /**
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
-     * @param $recommendedProducts
-     * @param $productsToDisplay
+     * @param mixed $productsToDisplayCounter
+     * @param mixed $limit
```
Comment (adeelq) [inline Block/Recommended/Product.php]: int

File: Block/Recommended/Product.php [hunk 41]
```diff
@@ -168,11 +174,13 @@ private function getProductsToDisplay($orderItems, $mode, &$productsToDisplayCou
     }
 
     /**
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
-     * @param $recommendedProducts
-     * @param $productsToDisplay
+     * @param mixed $productsToDisplayCounter
```
Comment (adeelq) [inline Block/Recommended/Product.php]: int

File: Block/Recommended/Product.php [hunk 42]
```diff
@@ -199,9 +207,9 @@ private function addRecommendedProducts(
     }
 
     /**
-     * @param $productsToDisplay
-     * @param $productsToDisplayCounter
-     * @param $limit
+     * @param mixed $productsToDisplay
```
Comment (adeelq) [inline Block/Recommended/Product.php]: array

File: Block/Recommended/Product.php [hunk 43]
```diff
@@ -199,9 +207,9 @@ private function addRecommendedProducts(
...
+     * @param mixed $productsToDisplayCounter
```
Comment (adeelq) [inline Block/Recommended/Product.php]: int

File: Block/Recommended/Product.php [hunk 44]
```diff
@@ -199,9 +207,9 @@ private function addRecommendedProducts(
...
+     * @param mixed $limit
```
Comment (adeelq) [inline Block/Recommended/Product.php]: int

File: Block/Recommended/Product.php [hunk 45]
```diff
@@ -228,8 +236,8 @@ private function fillProductsToDisplay($productsToDisplay, &$productsToDisplayCo
     /**
      * Product related items.
      *
-     * @param $productModel
-     * @param $mode
+     * @param mixed $productModel
```
Comment (adeelq) [inline Block/Recommended/Product.php]: \Magento\Catalog\Model\Product

File: Block/Recommended/Product.php [hunk 46]
```diff
@@ -228,8 +236,8 @@ private function fillProductsToDisplay($productsToDisplay, &$productsToDisplayCo
...
+     * @param mixed $mode
```
Comment (adeelq) [inline Block/Recommended/Product.php]: string

File: Block/Recommended/Quoteproducts.php [hunk 47]
```diff
@@ -107,11 +110,11 @@ public function getLoadedProductCollection()
     }
 
     /**
-     * @param $quoteItems
-     * @param $mode
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
+     * @param mixed $quoteItems
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: array

File: Block/Recommended/Quoteproducts.php [hunk 48]
```diff
@@ -107,11 +110,11 @@ public function getLoadedProductCollection()
...
+     * @param mixed $mode
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: string

File: Block/Recommended/Quoteproducts.php [hunk 49]
```diff
@@ -107,11 +110,11 @@ public function getLoadedProductCollection()
...
+     * @param mixed $productsToDisplayCounter
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: int

File: Block/Recommended/Quoteproducts.php [hunk 50]
```diff
@@ -107,11 +110,11 @@ public function getLoadedProductCollection()
...
+     * @param mixed $limit
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: int

File: Block/Recommended/Quoteproducts.php [hunk 51]
```diff
@@ -107,11 +110,11 @@ public function getLoadedProductCollection()
...
+     * @param mixed $maxPerChild
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: int

File: Block/Recommended/Quoteproducts.php [hunk 52]
```diff
@@ -145,11 +148,13 @@ private function getProductsToDisplay($quoteItems, $mode, &$productsToDisplayCou
...
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
-     * @param $recommendedProducts
-     * @param $productsToDisplay
+     * @param mixed $productsToDisplayCounter
+     * @param mixed $limit
+     * @param mixed $maxPerChild
+     * @param mixed $recommendedProducts
+     * @param mixed $productsToDisplay
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: array

File: Block/Recommended/Quoteproducts.php [hunk 53]
```diff
@@ -145,11 +148,13 @@ private function getProductsToDisplay($quoteItems, $mode, &$productsToDisplayCou
...
     /**
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
-     * @param $recommendedProducts
-     * @param $productsToDisplay
+     * @param mixed $productsToDisplayCounter
+     * @param mixed $limit
+     * @param mixed $maxPerChild
+     * @param mixed $recommendedProducts
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: array

File: Block/Recommended/Quoteproducts.php [hunk 54]
```diff
@@ -145,11 +148,13 @@ private function getProductsToDisplay($quoteItems, $mode, &$productsToDisplayCou
...
 
     /**
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
-     * @param $recommendedProducts
-     * @param $productsToDisplay
+     * @param mixed $productsToDisplayCounter
+     * @param mixed $limit
+     * @param mixed $maxPerChild
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: int

File: Block/Recommended/Quoteproducts.php [hunk 55]
```diff
@@ -145,11 +148,13 @@ private function getProductsToDisplay($quoteItems, $mode, &$productsToDisplayCou
     }
 
     /**
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
-     * @param $recommendedProducts
-     * @param $productsToDisplay
+     * @param mixed $productsToDisplayCounter
+     * @param mixed $limit
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: int

File: Block/Recommended/Quoteproducts.php [hunk 56]
```diff
@@ -145,11 +148,13 @@ private function getProductsToDisplay($quoteItems, $mode, &$productsToDisplayCou
     }
 
     /**
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
-     * @param $recommendedProducts
-     * @param $productsToDisplay
+     * @param mixed $productsToDisplayCounter
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: int

File: Block/Recommended/Quoteproducts.php [hunk 57]
```diff
@@ -176,9 +181,9 @@ private function addRecommendedProducts(
     }
 
     /**
-     * @param $productsToDisplay
-     * @param $productsToDisplayCounter
-     * @param $limit
+     * @param mixed $productsToDisplay
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: array

File: Block/Recommended/Quoteproducts.php [hunk 58]
```diff
@@ -176,9 +181,9 @@ private function addRecommendedProducts(
...
+     * @param mixed $productsToDisplayCounter
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: int

File: Block/Recommended/Quoteproducts.php [hunk 59]
```diff
@@ -176,9 +181,9 @@ private function addRecommendedProducts(
...
+     * @param mixed $limit
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: int

File: Block/Recommended/Quoteproducts.php [hunk 60]
```diff
@@ -205,8 +210,8 @@ private function fillProductsToDisplay($productsToDisplay, &$productsToDisplayCo
     /**
      * Product related items.
      *
-     * @param $productModel
-     * @param $mode
+     * @param mixed $productModel
+     * @param mixed $mode
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: string

File: Block/Recommended/Quoteproducts.php [hunk 61]
```diff
@@ -205,8 +210,8 @@ private function fillProductsToDisplay($productsToDisplay, &$productsToDisplayCo
     /**
      * Product related items.
      *
-     * @param $productModel
-     * @param $mode
+     * @param mixed $productModel
```
Comment (adeelq) [inline Block/Recommended/Quoteproducts.php]: \Magento\Catalog\Model\Product

File: Block/Recommended/Wishlistproducts.php [hunk 62]
```diff
@@ -163,11 +169,12 @@ public function getLoadedProductCollection()
     }
 
     /**
-     * @param $items
-     * @param $mode
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
+     * @param mixed $items
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: array

File: Block/Recommended/Wishlistproducts.php [hunk 63]
```diff
@@ -163,11 +169,12 @@ public function getLoadedProductCollection()
...
+     * @param mixed $mode
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: string

File: Block/Recommended/Wishlistproducts.php [hunk 64]
```diff
@@ -163,11 +169,12 @@ public function getLoadedProductCollection()
...
+     * @param mixed $productsToDisplayCounter
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: int

File: Block/Recommended/Wishlistproducts.php [hunk 65]
```diff
@@ -163,11 +169,12 @@ public function getLoadedProductCollection()
...
+     * @param mixed $limit
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: int

File: Block/Recommended/Wishlistproducts.php [hunk 66]
```diff
@@ -163,11 +169,12 @@ public function getLoadedProductCollection()
...
+     * @param mixed $maxPerChild
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: int

File: Block/Recommended/Wishlistproducts.php [hunk 67]
```diff
@@ -201,12 +208,14 @@ private function getProductsToDisplay($items, $mode, &$productsToDisplayCounter,
     }
 
     /**
-     * @param $productsToDisplayCounter
-     * @param $limit
-     * @param $maxPerChild
-     * @param $recommendedProducts
-     * @param $productsToDisplay
-     * @param $product
+     * @param mixed $productsToDisplayCounter
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: int

File: Block/Recommended/Wishlistproducts.php [hunk 68]
```diff
@@ -201,12 +208,14 @@ private function getProductsToDisplay($items, $mode, &$productsToDisplayCounter,
...
+     * @param mixed $limit
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: int

File: Block/Recommended/Wishlistproducts.php [hunk 69]
```diff
@@ -201,12 +208,14 @@ private function getProductsToDisplay($items, $mode, &$productsToDisplayCounter,
...
+     * @param mixed $maxPerChild
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: int

File: Block/Recommended/Wishlistproducts.php [hunk 70]
```diff
@@ -201,12 +208,14 @@ private function getProductsToDisplay($items, $mode, &$productsToDisplayCounter,
...
+     * @param mixed $recommendedProducts
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: array

File: Block/Recommended/Wishlistproducts.php [hunk 71]
```diff
@@ -201,12 +208,14 @@ private function getProductsToDisplay($items, $mode, &$productsToDisplayCounter,
...
+     * @param mixed $productsToDisplay
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: array

File: Block/Recommended/Wishlistproducts.php [hunk 72]
```diff
@@ -201,12 +208,14 @@ private function getProductsToDisplay($items, $mode, &$productsToDisplayCounter,
...
+     * @param mixed $product
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: \Magento\Catalog\Model\Product

File: Block/Recommended/Wishlistproducts.php [hunk 73]
```diff
@@ -234,9 +243,10 @@ private function addRecommendedProducts(
     }
 
     /**
-     * @param $productsToDisplay
-     * @param $productsToDisplayCounter
-     * @param $limit
+     * @param mixed $productsToDisplay
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: array

File: Block/Recommended/Wishlistproducts.php [hunk 74]
```diff
@@ -234,9 +243,10 @@ private function addRecommendedProducts(
...
+     * @param mixed $productsToDisplayCounter
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: int

File: Block/Recommended/Wishlistproducts.php [hunk 75]
```diff
@@ -234,9 +243,10 @@ private function addRecommendedProducts(
...
+     * @param mixed $limit
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: int

File: Block/Recommended/Wishlistproducts.php [hunk 76]
```diff
@@ -261,8 +271,8 @@ private function fillProductsToDisplay($productsToDisplay, &$productsToDisplayCo
     /**
      * Product related items.
      *
-     * @param $productModel
-     * @param $mode
+     * @param mixed $productModel
```
Comment (adeelq) [inline Block/Recommended/Wishlistproducts.php]: \Magento\Catalog\Model\Product

File: Block/Wishlist.php (hunk 1 above)
Comment (adeelq) [inline Block/Wishlist.php]: mixed or null|string|bool|int|\Magento\Store\Api\Data\StoreInterface

**Comment by adeelq** [comment]
all requested changes by me and @cdiacon are done.

**Comment by adeelq** [comment]
@cdiacon please merge this if you are satisfied with changes

**Pull request merged**